### PR TITLE
Some minor fixes

### PR DESCRIPTION
**Pull request by sudovijay**
- Better link parser
- Better error handling

### CODE REVIEW

**Comment by hosmelq** [comment]
Looks good to me.

I think you can make some changes in the Nav component, you are using state to manage the errors but you can use [Formik](https://github.com/jaredpalmer/formik#validate-values-values--formikerrorsvalues--promiseany) for that. Also maybe create a component for errors instead of `Name` to not use a `h2` tag for errors.

**Comment by sudovijay** [comment]
tbh its a pretty simple form why use formik for that? and ya could create a new component for errors but thats not required either it just form with 2 fields there ain't much error to show or catch,

**Comment by hosmelq** [comment]
If you don't want to use Formik i think you can remove the HOC. And i refer to creating a styled component to not use a h2 tag, for the semantic.

**Comment by sudovijay** [comment]
sure mate, on it 🙂 ,

**Comment by sudovijay** [comment]
@SaraVieira ~ hey, already quite a long PR, do look into wheneva you get time & lemme know if i need to change anything.. i quess im done here 🙂

**Comment by SaraVieira** [comment]
On it !!

File: Readme.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ The website was made with React, Styled Components, React-Apollo and the backend
...
..."https://avatars2.githubusercontent.com/u/417268?v=4" width="100px;"/><br /><sub><b>Pavithra Kodmad</b></sub>](http://pavithrakodmad.com)<br />[🐛](https://github.com/SaraVieira/awesome-talks/issues?q=author%3Apksjce "Bug reports") | [<img src="https://avatars0.githubusercontent.com/u/542140?v=4" width="100px;"/><br /><sub><b>James</b></sub>](https://jmes.tech)<br />[📦](#platform-varjmes "Packaging/porting to new platform") | [<img src="https://avatars0.githubusercontent.com/u/1051509?v=4" width="100px;"/><br /><sub><b>Sara Vieira</b></sub>](http://iamsaravieira.com)<br />[💻](https://github.com/SaraVieira/awesome-talks/commits?author=SaraVieira "Code") [🤔](#ideas-SaraVieira "Ideas, Planning, & Feedback") [🎨](#design-SaraVieira "Design") [📖](https://github.com/SaraVieira/awesome-talks/commits?author=SaraVieira "Documentation") | [<img src="https://avatars3.githubusercontent.com/u/8332043?v=4" width="100px;"/><br /><sub><b>Farzad YZ</b></sub>](http://farzadyz.com)<br />[💻](https://github.com/SaraVieira/awesome-talks/commits?author=farskid "Code") [🐛](https://github.com/SaraVieira/awesome-talks/issues?q=author%3Afarskid "Bug reports") [🤔](#ideas-farskid "Ideas, Planning, & Feedback") |
```
[the remark this answers is not in the document]
Comment (SaraVieira) [reply Readme.md]: Added you

File: src/Utils/link-parser.js [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
-        } else {
-            return null
-        }
-    } else {
-        const splitId = link.split('=')
-        return splitId.length === 2 ? splitId[1] : splitId[0]
+    let match = false
+
+    if (link.includes('v=') !== false) {
+        return link.substr(link.indexOf('v=') + 2, 11)
```
[the remark this answers is not in the document]
Comment (SaraVieira) [reply src/Utils/link-parser.js]: Changed it

File: src/Components/Nav.js [hunk 3]
```diff
@@ -77,12 +77,21 @@ injectGlobal`
 `
 
 const Name = styled.h2`
-    font-size: 400;
-    font-size: ${remcalc(22)};
+    font-weight: 400;
```
[the remark this answers is not in the document]
Comment (sudovijay) [reply src/Components/Nav.js]: it happens girl, you don't have to thanks me every time!

**Comment by SaraVieira** [comment]
I changed some small design stuff like: 
screen shot 2018-06-04 at 11 12 08

screen shot 2018-06-04 at 11 12 24

Also added moderator notes to the form, not required and will now merge and deploy!

File: Readme.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sudovijay) [reply Readme.md]: nah im fine, just don't like adding myself up there, there are already too many awesome peeps in the list to answer questions 🙂

**Comment by SaraVieira** [comment]
Thank you so much !!!! 🎉

Also added you as a colab :)

**Comment by sudovijay** [comment]
Hey, everything looks good i quess. need to work on my design skills lol . you're good in what you doing, just way fast .. and i was ok being a contributor, there was no need to add me as a collab.

**Comment by SaraVieira** [comment]
It's easier for both of us, you don't have to for and I don't have to download the project to test ahah

**Pull request merged**